UndertaleModLib/Util/QoiConverter.cs

```diff
@@ -63,19 +60,19 @@ public static Bitmap GetImageFromStream(Stream s)
         }
 
         /// <summary>
-        /// Creates a <see cref="Bitmap"/> from a <see cref="ReadOnlySpan{TKey}"/> of <see cref="byte"/>s.
+        /// Creates a raw format <see cref="GMImage"/> from a <see cref="ReadOnlySpan{TKey}"/> of <see cref="byte"/>s.
         /// </summary>
-        /// <param name="bytes">The <see cref="Span{TKey}"/> of <see cref="byte"/>s to create the PNG image from.</param>
-        /// <returns>The QOI image as a PNG.</returns>
+        /// <param name="bytes">The <see cref="Span{TKey}"/> of <see cref="byte"/>s to create the raw image from.</param>
+        /// <returns>The QOI image as a raw format image.</returns>
         /// <exception cref="Exception">If there is an invalid QOIF magic header or there was an error with stride width.</exception>
-        public static Bitmap GetImageFromSpan(ReadOnlySpan<byte> bytes) => GetImageFromSpan(bytes, out _);
+        public static GMImage GetImageFromSpan(ReadOnlySpan<byte> bytes) => GetImageFromSpan(bytes, out _);
 
         /// <summary><inheritdoc cref="GetImageFromSpan(System.ReadOnlySpan{byte})"/></summary>
         /// <param name="bytes"><inheritdoc cref="GetImageFromSpan(System.ReadOnlySpan{byte})"/></param>
         /// <param name="length">The total amount of data read from the <see cref="Span{TKey}"/>.</param>
         /// <returns><inheritdoc cref="GetImageFromSpan(System.ReadOnlySpan{byte})"/></returns>
         /// <exception cref="Exception"><inheritdoc cref="GetImageFromSpan(System.ReadOnlySpan{byte})"/></exception>
-        public unsafe static Bitmap GetImageFromSpan(ReadOnlySpan<byte> bytes, out int length)
+        public unsafe static GMImage GetImageFromSpan(ReadOnlySpan<byte> bytes, out int length)
```

Comment:
I think it might not be that bad, actually. At this point it's just looping over data in an array, so I guess the only performance gain we get from this is no bounds checks. Not sure how optimized the JIT is at getting rid of those, though.